openpype/hosts/max/plugins/publish/collect_review.py

```diff
@@ -54,8 +65,28 @@ def process(self, instance):
 
     @classmethod
     def get_attribute_defs(cls):
-
+        ocio_display_view_transform_list = ["sRGB||ACES 1.0 SDR-video"]
+        display_view_default = ""
+        if int(get_max_version()) >= 2024:
+            display_view_default = ""
+            ocio_display_view_transform_list = []
+            colorspace_mgr = rt.ColorPipelineMgr
+            displays = colorspace_mgr.GetDisplayList()
+            for display in sorted(displays):
+                views = colorspace_mgr.GetViewList(display)
+                for view in sorted(views):
+                    ocio_display_view_transform_list.append({
+                        "value": "||".join((display, view))
+                    })
+                    if display == "ACES" and view == "sRGB":
+                        display_view_default = "{0}||{1}".format(
+                            display, view
+                        )
```

Comment:
Can't we just label them like this:

```python
display_views = {}
if int(get_max_version()) >= 2024:
    colorspace_mgr = rt.ColorPipelineMgr
    for display in sorted(colorspace_mgr.GetDisplayList()):
        for view in sorted(colorspace_mgr.GetViewList(display)):
            display_views[(display, view)] = "{} || {}".format(display, view)
else:
    display = "sRGB"
    view = "ACES 1.0 SDR-Video"
    display_views[(display, view)] = "{} || {}".format(display, view)

# if the default value does not exist it is ignored by EnumDef
# and the first entry is taken instead
default_value = ("ACES", "sRGB")  
```

Where `items` for the EnumDef then is the `display_views` dict?
The resulting value from the enumdef would then also be that tuple.
So in the code you can then do: `display, view = creator_attributes["ocio_display_view_transform"]` if I'm not mistaken.

Reply:
I agree this method is more concise than with the list being in enumDef for this case (less lines of code),  but I have tried to pass the tuple as default_value it seems it seems the enum settings doesn't get it (let say the default value is ("ACES, "sRGB", but the setting doesn't show ACES || sRGB as default_value (the first value of the dict as the default instead)). 
So I stick with the list as the enum setting.